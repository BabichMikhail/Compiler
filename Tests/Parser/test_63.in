a[b.c.d] := d[b.c][b.a]