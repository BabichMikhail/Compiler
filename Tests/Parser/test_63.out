         a
      .
         b
   []
            c
         .
            b
      []
         d
:=
         d
      .
            c
         .
            b
   []
      a
